backend/sozisel/lib/sozisel_web/schema/queries/session_resource_queries.ex

```diff
@@ -18,6 +18,7 @@ defmodule SoziselWeb.Schema.Queries.SessionResourceQueries do
       @desc "Session's id"
       arg :id, non_null(:id)
       arg :token, non_null(:string)
+      arg :timestamp, :string
```

Comment:
don't we have `:datetime`?

Reply:
we have, but It's not stored or used anywhere in the backend. I've just added this argument as a workaround for not working `fetch-policy: network-only`. Thanks to that every time I pass a new timestamp to the request, I'm sure I'll get a response from the network and not from the cache.

Reply:
ugly as hell but if it works...